website/docs/Pipelines/Imputation_Pipeline/README.md

```diff
@@ -54,11 +64,17 @@ For examples of how to specify each input in a configuration file, as well as cl
  
 ### Imputation reference panel
  
-The reference panel files required for the Imputation workflow will soon be hosted in a public Google Bucket. See the [example input configuration](https://github.com/broadinstitute/warp/blob/develop/pipelines/broad/arrays/imputation/example_inputs.json) for the current reference panel files.
+The Imputation workflow's reference panel files is hosted in a [public Google Bucket](https://console.cloud.google.com/storage/browser/broad-gotc-test-storage/imputation/1000G_reference_panel;tab=objects?prefix=&forceOnObjectsSortingFiltering=false). For the cloud-path (URI) to the files, see the [example input configuration](https://github.com/broadinstitute/warp/blob/develop/pipelines/broad/arrays/imputation/example_inputs.json).
+
+#### Generation of the modified 1000 Genomes reference
+Initial tests of the Imputation workflow followed by assessments of polygenic risk score revealed that disease risk scores were lower when computed from imputed array data as opposed to whole-genome sequencing data, possibly because of incorrectly genotyped sites in the 1000 Genomes (1000G) reference panel. As a result, the 1000G reference files were modified for the Imputation pipeline as described below. You can view the original, unmodified 1000G VCFs [here](https://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/). Learn more about the modifications in this [overview](./references_overview.md)
```

Comment:
I would make the statement about the cause here stronger.  Instead of "... possibly because of incorrectly genotyped sites in the 1000G reference panel" I would say "This was found to be due to incorrectly genotyped sites in the 1000G reference panel."

Reply:
Also, say "as described below", but the description has all been moved in reference_overview.md

Reply:
Updated this.